### PR TITLE
Font, Hardcoded Languages & Server timeout fix

### PR DESCRIPTION
**Pull request by euanmillar**
This PR adds the default timeout to fix this issue that a Registrar cannot be created with any attachment on a slow connection:

https://github.com/opencrvs/opencrvs-core/issues/3426

fix-slow-network-upload

While testing on a slow connection I noticed that the font kept flashing and I realised it was being loaded on every render:

slow3g-webfont

So I decided to change the font loading process as I knew it was problematic.  I replaced the static font files in the public folder with https://github.com/typekit/webfontloader and now the fonts load from cache and dont flash on a slow connection.

slow3g-webfont-fixed

This should also fix: https://github.com/opencrvs/opencrvs-core/issues/3473

We used to allow different fonts for different languages which made configuring different languages impossible without an opencrvs-core change.  Now that we only use Noto Sans from Google, there was no need for this so I removed that requirement in the component library fonts.ts file.

Finally, I recognised a while ago that the hardcoded language select in Login and Client also prevents countries from implementing other languages without a core change. https://github.com/opencrvs/opencrvs-core/issues/2663 So I added a new config field in Farajaland repo for that.  Depends on: https://github.com/opencrvs/opencrvs-farajaland/pull/308

### CODE REVIEW

File: packages/login/src/views/StepTwo/StepTwoForm.tsx [hunk 1]
```diff
@@ -36,14 +36,18 @@ import {
...
 import { ceil } from 'lodash'
 import { messages } from '@login/i18n/messages/views/stepTwoForm'
 
 const StyledMobile2FA = styled(Mobile2FA)`
   transform: scale(0.8);
 `
+const StyledH2 = styled.h2`
+  ${({ theme }) => theme.fonts.h2};
+  /* stylelint-disable-next-line opencrvs/no-font-styles */
+  font-weight: 400;
```
Comment (euanmillar) [inline packages/login/src/views/StepTwo/StepTwoForm.tsx:49]: Please forgive.  I couldnt figure out why this h2 in the Login application is not rendering like the h2s in the client app.  Seems to force a bold weight somewhere and I ran out of time.

Comment (Zangetsu101) [reply packages/login/src/views/StepTwo/StepTwoForm.tsx:49]: Well the `h2` font defined here does have a font weight of 600. https://github.com/opencrvs/opencrvs-core/blob/53972be1befa72d0e0d24f337b69e28f6d52810c/packages/components/src/components/fonts.ts#L77

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

**Comment by Zangetsu101** [comment]
@euanmillar You might have missed this one: https://github.com/opencrvs/opencrvs-core/blob/9f5e2341bef42c2b37351b69f7801a75bb63a5d2/packages/components/src/components/charts/TriLineChart.tsx#L114

File: infrastructure/server-setup/example-1.ini [hunk 2]
```diff
@@ -9,11 +9,11 @@
 ; graphic logo are (registered/a) trademark(s) of Plan International.
 [docker-manager-first]
 ; Uncomment the line below
-; manager1 ansible_host="ENTER YOUR MANAGER HOST IP"
```
Comment (Zangetsu101) [inline infrastructure/server-setup/example-1.ini:12]: Was this added deliberately?

File: packages/login/src/views/StepTwo/StepTwoForm.tsx (hunk 1 above)
Comment (euanmillar) [reply packages/login/src/views/StepTwo/StepTwoForm.tsx:49]: Yes, but it doesnt seem to bold in the same way in the Login app.

Comment (euanmillar) [reply packages/login/src/views/StepTwo/StepTwoForm.tsx:49]: Just going to use a different style

Comment (euanmillar) [reply packages/login/src/views/StepTwo/StepTwoForm.tsx:49]: @Zangetsu101 this is the Login app without the hack:

Screenshot 2022-06-17 at 12 16 42

.. same header in client:

Screenshot 2022-06-17 at 12 15 44

**Pull request merged**